### ISSUE TITLE
Broken links for related projects

### ISSUE THREAD

**Issue by akshayka** [cvxpy/cvxpy#1503]
Several links on the related projects page (https://www.cvxpy.org/related_projects/index.html?highlight=dccp) are broken

They point to the cvxpy organization, but should instead point to cvxgrp.

**Comment by SteveDiamond** [comment]
Good catch! You should also change the pointer to cvxstoch to point to osmm.
